oscarlab/graphene
# Add musl support

## Description of the changes 

Disclaimer: musl is not binary-compatible with most programs, only the simplest can use musl binaries without recompilation. Others have to be compiled against musl headers (see [musl documentation](https://musl.libc.org/doc/1.1.24/manual.html)).

Resolves https://github.com/gramineproject/gramine/issues/49.

## How to test this PR? 

See the testing discussion on Reviewable.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/graphene/2269)
